locale/pt-br/texts.po

```diff
@@ -1244,6 +1244,31 @@ msgstr ""
 msgid "Custom network"
 msgstr "Rede personalizada"
 
+#: src/components/NanoContract/NanoContractsList.js:71
+msgid "Nano Contracts"
+msgstr "Nano Contracts"
+
+#: src/components/NanoContract/NoNanoContracts.js:22
+msgid "No Nano Contracts"
+msgstr "Nenhum Nano Contract"
+
+#: src/components/NanoContract/NoNanoContracts.js:24
+msgid ""
+"You can keep track of your registered Nano Contracts here once you have "
+"registered them."
+msgstr "Registre um Nano Contract e acompanhe neste tela."
+
+#: src/components/NanoContract/NoNanoContracts.js:29
+msgid "Learn More."
+msgstr "Saiba mais."
+
+#: src/components/NanoContract/RegisterNewNanoContractButton.js:22
+msgid "Register new"
+msgstr "Registrar novo token"
+
+#~ msgid "TOKENS"
```

Comment:
Can you just remove these? They are commented, right?

And also you renamed `TOKENS` to `tokens`

Reply:
Done.